Lib/importlib/util.py

```diff
@@ -244,5 +263,7 @@ def exec_module(self, module):
         loader_state = {}
         loader_state['__dict__'] = module.__dict__.copy()
         loader_state['__class__'] = module.__class__
+        loader_state['lock'] = threading.RLock()
+        loader_state['is_loading'] = threading.Event()
```

Comment:
Hi, can we use a `bool` flag instead of `threading.Event`? I see that access and modification to `loader_state['is_loading']` is protected by the `loader_state['lock']`, so there is no thread safety issue.

Using an additional `threading.Event` would introduce unnecessary resource costs.

Reply:
I agree, good catch. I will switch to a bool and push later today.